src/RingGrids/RingGrids.jl

```diff
@@ -9,50 +9,78 @@ import Statistics: mean
 import FastGaussQuadrature
 import LinearAlgebra
 
+# GPU
+import Adapt
+import KernelAbstractions
```

Comment:
KernelAbstractions is not used anywhere in RingGrids, isn't it?

Reply:
No, right, it was also added in `LowerTriangularMatrices` and I copied that over, will take it out.